src/locales/en.ts

```diff
@@ -898,6 +901,7 @@ export default {
       result_time: 'Time of Results',
       searching: 'Searching',
       distance: '{{distance}} {{unit}} away',
+      added_to_favorites: 'Added to Favorites',
```

Comment:
should we call this "follow" instead of "favorite"? something like "Added to followed hotspots" or "Followed hotspot"?

Reply:
oh, yeah, good catch.